package.json

```diff
@@ -54,7 +54,25 @@
     "prepack": "yarn lint && yarn test && yarn build",
     "happo": "happo run --config ./.happo.js",
     "happo-ci": "HAPPO_CONFIG_FILE='./.happo.js' npx -p happo.io happo-ci-github-actions --config ./.happo.js",
-    "contributors:add": "all-contributors add"
+    "contributors:add": "all-contributors add",
+    "postinstall": "simple-git-hooks",
+    "precommit:checks": "yarn test && yarn test:serverside --silent"
```

Comment:
I have a pretty heavy bias against pre commit or pre push checks that take a long time, especially if CI will do the longer running checks anyways.

In this case, I think we can just do the nano-staged formatting and linting automatically and leave it to CI to run the tests